### DISCUSSION TITLE
LosslessCut clone?

### DISCUSSION THREAD

**Discussion by kubalav** [mifi/lossless-cut#1577] [General]
This software is too similar to LosslessCut. It costs $24.99. Is that ok?

[PazuVideoCutter](https://www.pazusoft.com/video-cutter-for-windows/)

**Comment by mifi** [comment]
Thanks a lot for reporting! It does look like a shameless clone indeed. And I don't see the GPL license statement anywhere so it should be an illegal GPL violation. I'll add it here: #550
